docs/development/machine-bootstrap.md

```diff
@@ -0,0 +1,19 @@
+# Machine-Controller integration into Gardener
+The Gardener is deploying the [machine-controller-manager](https://github.com/gardener/machine-controller-manager) as part of a Shoot controlplane onto a Seed to manage the nodes/machines of the Shoot cluster. The machine-controller-manager will create, bootstrap, update and delete the nodes of the Shoot cluster and the underlying virtual machines.
+
+## Flow of creating and bootstrapping Machines/Nodes
+The gardener-controller-manager will ensure during the creation/reconciliation flow
```

Comment:
s / 
The gardener-controller-manager will ensure during the creation/reconciliation flow /
The following steps will be ensured by the machine-controller-manager during a creation/reconciliation